spec/hosts/nodes/konkong01.ls.lsst.org_spec.rb

```diff
@@ -80,20 +86,48 @@
         end
       end
 
-      context 'with enp129s0f0' do
-        let(:interface) { 'enp129s0f0' }
+      %w[
+        enp129s0f0
+        enp129s0f1
+      ].each do |i|
+        context "with #{i}" do
+          let(:interface) { i }
+
+          it_behaves_like 'nm enabled interface'
+          it_behaves_like 'nm ethernet interface'
+          it_behaves_like 'nm bond slave interface', master: 'bond0'
+        end
+      end
+
+      context 'with bond0' do
+        let(:interface) { 'bond0' }
 
         it_behaves_like 'nm enabled interface'
-        it_behaves_like 'nm dhcp interface'
-        it_behaves_like 'nm ethernet interface'
+        it_behaves_like 'nm bond interface'
+        it_behaves_like 'nm no-ip interface'
+        it { expect(nm_keyfile['bond']['xmit_hash_policy']).to eq('layer3+4') }
```

Comment:
Could we move this into the `nm bond interface` shared example and then update the rspec for elqui and pillan?  The only other bond configuration in-tree appears to be in `hieradata/cluster/amor/osfamily/RedHat/major/7.yaml` and that file can/should be removed.

Reply:
thanks so much! i will resolve the tests and we are good to merge